docker/develop/run

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set +x
```

Comment:
Do we need this still? Personally I prefer scripts to be silent other than logging through `echo`s.

Reply:
Sorry, left from debug